src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelector.js

```diff
@@ -174,11 +204,8 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
                     this.checkValidity();
```

Comment:
is this one still needed?

Reply:
This is still needed for the case where there is `internalStateEditor` -- at least that is unless the consumer of the component uses it in the same way as `ApprovalCommentForm` which takes the `editorState` returned from `onChange` and sets it back as the prop on `DraftJSMentionSelector`